quinn-proto/src/connection/transmit_buf.rs

```diff
@@ -193,23 +196,3 @@ impl<'a> TransmitBuf<'a> {
         self.buf.as_mut_slice()
     }
 }
-
```

Comment:
It would be easier to review with our development workflow if you edited your PR history to undo this addition rather than including a commit that adds these lines and a subsequent one that removes them